### PR TITLE
initcap support

### PR DESCRIPTION
**Pull request by derekmorr**
This addressed #1177

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by williamBlazing** [comment]
add to whitelist

File: engine/src/parser/expression_utils.cpp [hunk 1]
```diff
@@ -54,6 +54,7 @@ bool is_unary_operator(operator_type op) {
 	case operator_type::BLZ_CHAR_LENGTH:
 	case operator_type::BLZ_STR_LOWER:
 	case operator_type::BLZ_STR_UPPER:
+        case operator_type::BLZ_STR_INITCAP:
```
Comment (williamBlazing) [inline engine/src/parser/expression_utils.cpp:57]: please  fix spacing so that it matches the rest

File: engine/src/parser/expression_utils.cpp [hunk 2]
```diff
@@ -135,6 +136,7 @@ cudf::type_id get_output_type(operator_type op, cudf::type_id input_left_type) {
 		return cudf::type_id::TIMESTAMP_NANOSECONDS;
 	case operator_type::BLZ_STR_LOWER:
 	case operator_type::BLZ_STR_UPPER:
+        case operator_type::BLZ_STR_INITCAP:
```
Comment (williamBlazing) [inline engine/src/parser/expression_utils.cpp:139]: please  fix spacing so that it matches the rest

File: engine/src/parser/expression_utils.cpp [hunk 3]
```diff
@@ -285,6 +287,7 @@ operator_type map_to_operator_type(const std::string & operator_token) {
 		{"CHAR_LENGTH", operator_type::BLZ_CHAR_LENGTH},
 		{"LOWER", operator_type::BLZ_STR_LOWER},
 		{"UPPER", operator_type::BLZ_STR_UPPER},
+                {"INITCAP", operator_type::BLZ_STR_INITCAP},
```
Comment (williamBlazing) [inline engine/src/parser/expression_utils.cpp:290]: please  fix spacing so that it matches the rest

File: engine/src/parser/expression_utils.hpp [hunk 4]
```diff
@@ -46,6 +46,7 @@ enum class operator_type {
 	BLZ_CHAR_LENGTH,
 	BLZ_STR_LOWER,
 	BLZ_STR_UPPER,
+        BLZ_STR_INITCAP,
```
Comment (williamBlazing) [inline engine/src/parser/expression_utils.hpp:49]: please  fix spacing so that it matches the rest

**Review by williamBlazing — CHANGES_REQUESTED** [review]

**Comment by williamBlazing** [comment]
Please update the CHANGELOG so that it triggers CI

File: engine/src/parser/expression_utils.hpp (hunk 4 above)
Comment (derekmorr) [reply engine/src/parser/expression_utils.hpp]: addressed

File: engine/src/parser/expression_utils.cpp (hunk 3 above)
Comment (derekmorr) [reply engine/src/parser/expression_utils.cpp]: addressed

File: engine/src/parser/expression_utils.cpp (hunk 2 above)
Comment (derekmorr) [reply engine/src/parser/expression_utils.cpp]: addressed

File: engine/src/parser/expression_utils.cpp (hunk 1 above)
Comment (derekmorr) [reply engine/src/parser/expression_utils.cpp]: addressed

**Comment by derekmorr** [comment]
Changes should be addressed.

**Comment by williamBlazing** [comment]
rerun-tests

**Comment by williamBlazing** [comment]
It seems like a couple queries are failing to match against their results. Looking at the logs it almost seems like the results from drill are the ones that are incorrect. Its worth digging in more to see if that is the case. It would not be the first time we run into something like that

**Comment by Christian8491** [comment]
Hi @derekmorr  could you please review the last two comments in order to fix these issues related to gpuCI jobs.

**Pull request closed without merging**